bin/generate-bundle.js

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env node
+const glob = require('glob');
+const path = require('path');
+const less = require('less');
+const fs = require('fs');
+const pkg = require('../package.json');
+const LessPluginCleanCSS = require('less-plugin-clean-css');
+const cleanCSSPlugin = new LessPluginCleanCSS({ advanced: true });
+const LessPluginAutoPrefix = require('less-plugin-autoprefix');
+const autoprefixPlugin = new LessPluginAutoPrefix();
+const currentDir = path.dirname(__dirname);
+const { exec } = require('child_process');
+
+// The list of directories in the dist to pull
+const dsList = ['ds4', 'ds6'];
+
+/**
+ * Main Processing class. Holds info about args passed and ds version
+ */
+class CssProcesser {
+  constructor(dsV, args) {
```

Comment:
Can we use dsVersion rather than dsV, so its clear what it is, unless "dsV" is a common thing in the repo.